### ISSUE TITLE
attach binaries to Github releases

### ISSUE THREAD

**Issue by yegor256** [yegor256/rultor#662]
Let's make it possible to attach binaries to Github releases. For example, in `.rultor.yml`:

```yaml
release:
  artifacts:
    - "target/rultor-${tag}.jar"
```

Rultor should get that `target/rultor-${tag}.jar` file and attach it as a binary artifact to the Github release created.

**Comment by alex-palevsky** [comment]
@wentwog the task is your hands for the last 8 days.. keep in mind that if it's not closed in the next 40 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle. -5 added to your rating, current score is: +20

**Comment by wentwog** [comment]
@alex-palevsky I understand. Sorry. Better give this task to someone else since I feel I won't be able to meet the deadline.

**Comment by alex-palevsky** [comment]
> @alex-palevsky I understand. Sorry. Better give this task to someone else since I feel I won't be able to meet the deadline.

@wentwog right, I will find someone else, no problem

**Comment by alex-palevsky** [comment]
@dpisarenko the task is yuors now go ahead

**Comment by alex-palevsky** [comment]
@dpisarenko this ticket is not yours any more, thanks

**Comment by alex-palevsky** [comment]
@alevohin it's yours now, please proceed keeping in mind [our principles](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Feel free to ask any technical questions right here in the ticket. The cost of this task is **30 mins** (this is exactly how much will be paid, not less not more), when the task is [done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by alevohin** [comment]
@alex-palevsky could you assing this ticket for someone else? I have OS Windows on my workstation, unit test fails under Windows.

**Comment by alex-palevsky** [comment]
> @alex-palevsky could you assing this ticket for someone else? I have OS Windows on my workstation, unit test fails under Windows.

@alevohin OK, I will try to assign someone else

**Comment by alex-palevsky** [comment]
@wb14123 This task is yours, please go ahead keeping in mind [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). If any questions, don't hesitate to ask right here... The budget of this task is **30 mins**. This is exactly how much time will be compensated, when the task is completed. Read about our [Definition of Done](http://www.xdsd.org/2014/04/15/definition-of-done.html)

**Comment by alex-palevsky** [comment]
@wb14123 you're working with this ticket for the last 16 days. If it's not closed within the next 48 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle. [This article](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html) should help if you're stuck; -30 to your rating, your total score is -15

**Comment by alex-palevsky** [comment]
@wb14123 it takes too long. Usually, we expect any task to be finished in less than a week. I'll assign someone else. Please stop working with it right now. See our [no obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle. added -60 to your rating, now it is equal to -345

**Comment by alex-palevsky** [comment]
@krzyk this is your task now, help us

**Comment by krzyk** [comment]
PR #769

**Comment by alex-palevsky** [comment]
there is a puzzle in this code `662-8cc0c29a`, we'll resolve it later

**Comment by krzyk** [comment]
@yegor256 please close this issue, it has been merged to master

**Comment by yegor256** [comment]
@krzyk thanks!

**Comment by alex-palevsky** [comment]
@krzyk Thanks so much! Your account was topped up for **30 mins** (transaction ID is `52107396`). +30 to your rating, your total score is [+4538](http://www.netbout.com/b/35035?open=rating)

**Comment by jeromebridge** [comment]
This doesn't seem to be working for me.  Here is the configuration I'm using:
````
install: |-
  sudo apt-get -q -y install lib32z1 lib32ncurses5 lib32bz2-1.0
release:
  pre: false
  artifacts:
    - "inspro-fitnesse-installer/dist/inspro-fitnesse-installer-${tag}.exe"
  script: |
    mvn versions:set "-DnewVersion=${tag}" -DartifactId=* -DgroupId=* -DoldVersion=*
    git commit -am "${tag}"
    mvn clean deploy --settings documentation/files/maven-settings.xml -Prelease
````
Here is what Rultor says it knows:
````
<?xml version="1.0" encoding="UTF-8"?>
<p>
   <entry key="install">sudo apt-get -q -y install lib32z1 lib32ncurses5 lib32bz2-1.0</entry>
   <entry key="release">
      <entry key="pre">false</entry>
      <entry key="artifacts">
         <item>inspro-fitnesse-installer/dist/inspro-fitnesse-installer-${tag}.exe</item>
      </entry>
      <entry key="script">mvn versions:set "-DnewVersion=${tag}" -DartifactId=* -DgroupId=* -DoldVersion=*
git commit -am "${tag}"
mvn clean deploy --settings documentation/files/maven-settings.xml -Prelease</entry>
   </entry>
</p>
````
Here is my <a href="http://www.rultor.com/t/4908-133718025">log output</a> at the end of the release:
````
+ mv /home/r/repo . 
++ whoami 
+ chown -R root repo 
+ '[' -n '' ']' 
++ whoami 
+ sudo chown -R rultor repo 
+ cd repo 
+ git checkout __rultor Already on '__rultor' 
+ git tag 1.00.019 -m '1.00.019: tagged by rultor.com' 
+ git reset --hard HEAD is now at 2b17609 1.00.019 
+ git clean -fd 
+ git checkout master Switched to branch 'master' 
+ git branch -D __rultor Deleted branch __rultor (was 2b17609). 
+ git push --all origin Everything up-to-date 
+ git push --tags origin To git@github.com:jeromebridge/inspro-integration.git * [new tag]	1.00.019 -> 1.00.019
````

**Comment by alex-palevsky** [comment]
@yegor256 last puzzled solved `662-8cc0c29a`/#832

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#1371](https://github.com/yegor256/rultor/issues/1371) is still not solved.
